### PR TITLE
What have the Romans ever done for us?

### PR DESCRIPTION
**Pull request by SimonDarksideJ**
Except for:

* Roads
* Health
[x] Massive folder reorgs
[x] Pending namespace changes
* Peace?

Pah

### CODE REVIEW

**Comment by davidkline-ms** [comment]
You forgot the aquaducts! :)

File: Assets/MixedRealityToolkit/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Mixed Reality Toolkit
+
+This folder contains all the individual features of the MRTK, providing systems to assist in VR/XR/AR solutions
+The Core folder is required and should not be removed, whereas the other feature folders are completely optional. Wither use the versions provided by the toolkit, or roll your own.
+
+# In this Volder
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/README.md]: Volder -> Folder

File: Assets/MixedRealityToolkit/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+## _Core
+
+The Core folder contains all the internal functions and utilities nessasary for the operation of the Mixed Reality Toolkit.  
+
+> ### THIS FOLDER SHOULD NOT BE REMOVED
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/README.md]: should to cannot?

File: Assets/MixedRealityToolkit-SDK/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Mixed Reality Toolkit - SDK
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit-SDK/README.md]: Weren't we going to have < Feature > folders in SDK as well?

Comment (davidkline-ms) [reply Assets/MixedRealityToolkit-SDK/README.md]: MixedRealityToolkit <- Pure C# (Mostly)
       /Required
       /< Features > <- Optional
   MixedRealityToolkit-SDK < Optiona l> <- Monobehaviours
       /< Features >

File: Assets/MixedRealityToolkit/README.md (hunk 2 above)
Comment (davidkline-ms) [reply Assets/MixedRealityToolkit/README.md]: maybe even stronger... MUST NOT?

File: Assets/MixedRealityToolkit/_Core/README.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Mixed Reality Toolkit - Core
+
+This folder contains all the MRTK Internal components / functionality which should not be removed from the project.
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit/_Core/README.md]: should -> cannot or must?

File: Assets/MixedRealityToolkit-SDK/README.md (hunk 3 above)
Comment (SimonDarksideJ) [reply Assets/MixedRealityToolkit-SDK/README.md]: The SDK folder can stay as it is for now until we start adding components.  A feature for a later sprint :D

File: Assets/MixedRealityToolkit/README.md (hunk 1 above)
Comment (SimonDarksideJ) [reply Assets/MixedRealityToolkit/README.md]: Rofl, my faux German filter must have kicked in :D

File: ProjectSettings/ProjectSettings.asset [hunk 5]
```diff
@@ -619,32 +598,34 @@ PlayerSettings:
...
-  scriptingRuntimeVersion: 0
+  scriptingRuntimeVersion: 1
   apiCompatibilityLevelPerPlatform:
     Metro: 3
   m_RenderingPath: 1
   m_MobileRenderingPath: 1
-  metroPackageName: MixedRealityToolkit-Unity
-  metroPackageVersion: 2017.3.1.0
+  metroPackageName: Microsoft.MixedReality.Toolkit
+  metroPackageVersion: 1.0.0.0
```
Comment (StephenHodgson) [inline ProjectSettings/ProjectSettings.asset]: What was the first version we agreed on?

Comment (SimonDarksideJ) [reply ProjectSettings/ProjectSettings.asset]: Darn it, in one update I changed it to 2018.1.0 Which seems to have not stuck. 
DO we want to change this again for this PR?

Comment (StephenHodgson) [reply ProjectSettings/ProjectSettings.asset]: It's not important

File: Assets/MixedRealityToolkit-SDK/README.md (hunk 3 above)
Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-SDK/README.md]: Agreed

File: ProjectSettings/ProjectSettings.asset (hunk 5 above)
Comment (davidkline-ms) [reply ProjectSettings/ProjectSettings.asset]: Not important now. I expect we will use yymm.release.patch or yyyy.mm.patch when we hit alpha.

**Pull request merged**